src/blocks/skeleton/Skeleton.types.ts

```diff
@@ -0,0 +1,70 @@
+import { CSSProperties, ReactNode } from 'react';
+
+import { BlocksColors, ResponsiveProp, SpaceType, ValueOf } from '../Blocks.types';
+import { FlattenSimpleInterpolation } from 'styled-components';
+
+export type SkeletonResponsiveProps = {
+  /* Sets display css property */
+  display?: ResponsiveProp<CSSProperties['display']>;
```

Comment:
display not required as nothing is going to be rendered and visible inside skeleton component.